packages/app/src/elements/import-dialog.ts

```diff
@@ -20,15 +26,66 @@ export class ImportDialog extends Dialog<File, void> {
     @state()
     private _items: VaultItem[] = [];
 
+    @state()
+    private _itemColumns: imp.ImportCSVColumn[] = [];
+
+    @query("#csvHasDataOnFirstRowCheckbox")
+    private _csvHasDataOnFirstRowCheckbox: HTMLInputElement;
+
     @query("#formatSelect")
     private _formatSelect: Select<string>;
 
     @query("#vaultSelect")
     private _vaultSelect: Select<Vault>;
 
+    @query("#nameColumnSelect")
+    private _nameColumnSelect: Select<number>;
+
+    @query("#tagsColumnSelect")
+    private _tagsColumnSelect: Select<number>;
+
+    @queryAll("pl-select.field-type-select")
+    private _fieldTypeSelects: Select<FieldType>[];
+
+    @queryAll("pl-input.field-name-input")
+    private _fieldNameInputs: Input[];
+
+    static styles = [
+        ...Dialog.styles,
+        css`
+            :host {
+                --pl-dialog-max-width: 40em;
+            }
+        `,
+    ];
+
+    // TODO: Fix dialog reset after closed (when re-opened)
```

Comment:
@MaKleSoft do you have any idea on where should I look at? I saw some `reset` functions in other elements but I haven't investigated yet on what's that about, etc.

Reply:
Anything needing resetting should be done in the `show()` method.

Reply:
In this case you should reset `this._itemColumns` and the value of `this._csvHasDataOnFirstRowCheckbox`.